catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/DataListFragment.java

```diff
@@ -244,18 +244,16 @@ protected void finishActionMode() {
 	}
 
 	public void showDeleteAlert(final List<UserData> selectedItems) {
-		new AlertDialog.Builder(getContext())
-				.setTitle(R.string.deletion_alert_title)
-				.setMessage(R.string.deletion_alert_text)
-				.setPositiveButton(R.string.deletion_alert_yes, new DialogInterface.OnClickListener() {
-					@Override
-					public void onClick(DialogInterface dialog, int id) {
-						deleteItems(selectedItems);
-					}
-				})
-				.setNegativeButton(R.string.no, null)
-				.setCancelable(false)
-				.show();
+		if (isItemUsed(selectedItems)) {
```

Comment:
Right, I just replicated the behavior of the function which it had before editing, i.e. deleting everything, even if items are in use, though now there is no warning about that. And also we suppose that the deletion should be prohibited even if only one of the selected items is in use. I talked about showing a list of used user items in the warning message with Wolfgang and he said that it could be done in the next iteration of the ticket, so yeah, during that iteration user should only have permission to delete selected items while all of them are not in use.